actors/builtin/market/market_actor.go

```diff
@@ -2,6 +2,7 @@ package market
 
 import (
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
FYI we are moving away from `pkg/errors`, don't introduce more use of it. Use `xerrors`.

Reply:
We don't this anymore.